recipes/snmf/meta.yaml

```diff
@@ -0,0 +1,26 @@
+about:
+  home: https://github.com/avengerpb/snmf_conda
+  license: Artistic-2.0
+  summary: Estimating individual admixture coefficients based on sparse non-negative matrix factorization and population genetics
+
+build:
+  number: 0
+  skip: False
+
+requirements:
+  build:
+    - perl-bioperl
+    - perl-lib
+
+package:
+  name: snmf
+  version: 1.0.0
+
+source:
+   fn: 1.5.tar.gz
+   url: https://github.com/avengerpb/snmf_conda/archive/1.5.tar.gz
```

Comment:
Yep, if you check out some of the java packages, the wrapper script has to handle that kind of use case, e.g. https://github.com/bioconda/bioconda-recipes/blob/master/recipes/gatk/3.5/gatk.sh.
But I'm not sure we need to symlink if you do it that way I suggested -- everything can go in the bin dir. @bgruening had suggested it in https://github.com/bioconda/bioconda-recipes/pull/5492#discussion_r133381946, but at that time I don't think it was clear what the recipe was doing.